doc/lua_api.txt

```diff
@@ -6101,6 +6109,23 @@ object you are working with still exists.
 * `hud_set_hotbar_selected_image(texturename)`
     * sets image for selected item of hotbar
 * `hud_get_hotbar_selected_image`: returns texturename
+* `set_minimap_modes({mode, mode, ...}, selected_mode)`
+    * Overrides the available minimap modes and changes the selected mode.
+      `mode` is a table consisting of up to four fields:
+    * `type`: Available type:
```

Comment:
all of the below needs extra level of indentation to indicate this is a child of the above